Docu: Add julia wrapper and contributions in Readme

JoKaBus opened the pull request.

JoKaBus committed:
Docu: Add julia wrapper and contributions in Readme

JoKaBus commented on README.md:
````diff
@@ -229,6 +247,40 @@ In the `examples/python/` folder, you can find two more Python examples:
+# Development / latest GitHub version (optional)
+# Pkg.add(url="https://github.com/epsteinlib/EpsteinLib.jl")
+```
+The Julia wrapper is developed in a separate repository ([epsteinlib/EpsteinLib.jl](https://github.com/epsteinlib/EpsteinLib.jl)) by Jonathan K. Busse (@JoKaBus) and David Gómez-Castro (@dgomezcastro).
````
What do you think about the contribution statement here? @dgomezcastro
The rest of the text here is copied with minor changes from the other repo.

dgomezcastro commented:
In order to avoid duplicating the example (which usually leads to one of them being outdated), you can simply point to the MWE in the `EpsteinLib.jl` repo.

JoKaBus committed:
Feature: Add minimal julia example

JoKaBus committed:
Docu: minor spelling fixes, removed 1 readme sentence thats not adding anything new

JoKaBus committed:
Docu: mention julia example in readme

JoKaBus commented:
> In order to avoid duplicating the example (which usually leads to one of them being outdated), you can simply point to the MWE in the `EpsteinLib.jl` repo.

Good point about the duplication issue. I think its fine here, as we are already duplicating all the other examples (mathematica, python, c). I think we should duplicate all the minimal examples, or none of them.

But i added a sentence where the example can be found in `examples/julia/lattice_sum.jl` to make this clear.

JoKaBus committed:
Docu: Example in changelog

dgomezcastro commented:
but all the mathematica, python, c code is in this repo. So it is somehow more natural to have the examples here. Having them twice in the README and somewhere could be redundant, but I guess it may also be a convention.

`lattice_sum.jl` works in my machine. I, however, would rather not approve a review that copies the README from another repo into this repo. I suggest linking to the README in the `EpsteinLib.jl`. 

In summary, I do not oppose this pull request because I can understand the convention, but I would prefer not to actively approve it. I also point out that I am not the reviewer.

JoKaBus committed:
Docu: link author names for julia wrapper

JoKaBus commented:
> In summary, I do not oppose this pull request because I can understand the convention, but I would prefer not to actively approve it. I also point out that I am not the reviewer.

Thanks @dgomezcastro , i appreciate the input, even if you are not a reviewer! Yes, this makes total sense. I think linking the repo as you suggested is also a good choice, i will let @andreasbuchheit decide on this.

As a last change, i put your name in the contribution mention first and linked the github profiles, as `@dgomezcastro` does not seem to work in `.md` files.

JoKaBus committed:
Docu: linke julia repository for installation instructions

Updated the README to include information about the Julia library EpsteinLib.jl and removed the previous installation instructions and example code.

JoKaBus committed:
Docu: grammar readme

JoKaBus committed:
Docu: julia wrapper not library

JoKaBus commented:
Decided to follow @dgomezcastro suggestion and link to the Julia repo in the `in Julia` section. Thanks again for the feedback!
